qcelemental/dft_functional_info.py

```diff
@@ -10,7 +10,10 @@
 
 
 class DFTFunctional(ProtoModel):
-    ansatz: int = Field(..., description="SuperFunctional rung.")
+    ansatz: int = Field(
+        ...,
+        description="SuperFunctional rung (is what Psi4 claims this is. In actuality, this is 2 for meta-GGAs, 1 for GGAs and 0 otherwise.)",
```

Comment:
"The required derivative of the density for the functional. (LDA = 0, GGA = 1, MGGA=2).